firefox-ios/Storage/Rust/RustRemoteTabs.swift

```diff
@@ -6,19 +6,16 @@ import Foundation
 import Shared
 import Common
 
-import class MozillaAppServices.TabsStorage
+import class MozillaAppServices.TabsStore
 import enum MozillaAppServices.TabsApiError
 import struct MozillaAppServices.ClientRemoteTabs
 import struct MozillaAppServices.RemoteTabRecord
 
 public class RustRemoteTabs {
     let databasePath: String
-
     let queue: DispatchQueue
-    var storage: TabsStorage?
-
+    var store: TabsStore?
```

Comment:
This commit is stacked on my TabsStore Refactor, which will land before this: https://github.com/mozilla-mobile/firefox-ios/pull/20776